assets/js/modules/adsense/components/dashboard/DashboardTopEarningPagesWidgetGA4.js

```diff
@@ -102,6 +125,7 @@ DashboardTopEarningPagesWidgetGA4.propTypes = {
 	WidgetNull: PropTypes.elementType.isRequired,
 };
 
-export default whenActive( { moduleName: 'adsense' } )(
-	DashboardTopEarningPagesWidgetGA4
-);
+export default compose(
+	whenActive( { moduleName: 'adsense' } ),
+	whenActive( { moduleName: 'analytics-4' } )
```

Comment:
@aaemnnosttv, we also need to update the storybook story to have `analytics-4` active there because without it the widget doesn't render there: https://google.github.io/site-kit-wp/storybook/pull/6844/?path=/story/modules-adsense-widgets-dashboardtopearningpageswidgetga4--ready

Reply:
Ah good catch, thanks 👍

Reply:
Should be good now 👍 
<img width="1062" alt="image" src="https://user-images.githubusercontent.com/1621608/230929417-76d7dd9d-58ea-4f93-8e3c-625103e1ed8b.png">

Reply:
Great, thanks!